### PR TITLE
feat(python): Add Flask app example

### PR DESCRIPTION
**Pull request by izzymiller**
I find that despite being questionable for production, Flask is hands down the easiest way to stand up a complete Python application. I figured we ought to have an easy to dissect example of Flask and Looker working in concert.

This PR effectively clones the canonical [Flask tutorial](https://flask.palletsprojects.com/en/1.1.x/tutorial/) and adds some very lightweight Looker functionality that's easy to understand and extend.

It's pretty lightweight and self explanatory but let me know if you think I should better document the code, or change anything structurally.

## Developer Checklist [ℹ️](https://github.com/looker-open-source/sdk-codegen/blob/main/CONTRIBUTING.md#developer-checklist)

**most do not apply as this is just an example**
- [ ] ♿️ Accessibility addressed
- [ ] 🌏 Localization & Internationalization addressed
- [ ] 🖼 Image Snapshot coverage
- [ ] 📚 Documentation updated
- [ ] 🧳 Bundle size impact addressed
- [ ] 🏁 Performance impacts addressed
- [X] 👾 Browsers tested
    - [X] Chrome
    - [x] Firefox
    - [X] Safari
    - [X] IE11

### CODE REVIEW

File: examples/python/lookersdk-flask/app/__init__.py [hunk 1]
```diff
@@ -0,0 +1,33 @@
+import os
+
+from flask import Flask
+from . import db
+from . import auth
+from . import blog
+from . import looker
```
Comment (joeldodge79) [inline examples/python/lookersdk-flask/app/__init__.py:7]: unused?

File: examples/python/lookersdk-flask/app/__init__.py [hunk 2]
```diff
@@ -0,0 +1,33 @@
+import os
+
+from flask import Flask
```
Comment (joeldodge79) [inline examples/python/lookersdk-flask/app/__init__.py:3]: [the goog](https://g3doc.corp.google.com/eng/doc/devguide/py/style/index.md?cl=head#s2.2-imports) wants it like this: `import flask` (see suggested change below)

File: examples/python/lookersdk-flask/app/__init__.py [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+from . import db
+from . import auth
+from . import blog
+from . import looker
+
+
+def create_app(test_config=None):
+    app = Flask(__name__, instance_relative_config=True)
```
Comment (joeldodge79) [inline examples/python/lookersdk-flask/app/__init__.py:11]:
```suggestion
    app = flask.Flask(__name__, instance_relative_config=True)
```

File: examples/python/lookersdk-flask/app/auth.py [hunk 4]
```diff
@@ -0,0 +1,96 @@
...
+    render_template,
+    request,
+    session,
+    url_for,
+    current_app,
+)
+from werkzeug.security import check_password_hash, generate_password_hash
+
+from .db import get_db
+from .looker import get_my_user
```
Comment (joeldodge79) [inline examples/python/lookersdk-flask/app/auth.py:17]: ditto on import style here and etc

**Review by joeldodge79 — CHANGES_REQUESTED** [review]
nice work @izzymiller - just a couple style nits but otherwise LGTM

File: examples/python/lookersdk-flask/app/__init__.py (hunk 1 above)
Comment (izzymiller) [reply examples/python/lookersdk-flask/app/__init__.py:7]: Need em all except looker, but good callout.

**Review by joeldodge79 — APPROVED** [review]

**Comment by josephaxisa** [comment]
I am late to the party but wanted to say thanks for putting this together, looks great!

**Pull request merged**